src/Robot/RobotsTxt.php

```diff
@@ -53,4 +53,20 @@ public function isAllowed($userAgent, $path)
         $urlParts = array_filter(explode('/', $path));
         return $this->tree->allowed(strtolower($userAgent), $urlParts) !== false;
     }
+
+    /**
+     * Is the given user agent disallowed access to the given resource?
+     * @param  string  $userAgent The user agent to check
+     * @param  string  $path The path of the url
+     * @return bool|null
+     */
+    public function isDisallowed($userAgent, $path) 
+    {
+        $status = $this->isAllowed($userAgent, $path);
```

Comment:
Thanks for contributing!
I think you can simplify this function to the following

```php
public function isDisallowed($userAgent, $path) 
{
    return !$this->isAllowed($userAgent, $path);
}
```